app/views/saml_idp/shared/saml_post_binding.html.erb

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
   <head>
     <meta charset="utf-8" />
     <title><%= t('headings.redirecting') %> | <%= APP_NAME %></title>
+    <%= javascript_tag(nonce: true) do %>
+      document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/, 'js');
```

Comment:
could we assume `classList` is supported? I think all our officially supported browsers do right?
```suggestion
      document.documentElement.classList.remove('no-js');
      document.documentElement.classList.add('no-js');
```

Reply:
Yeah, that's a good point. At first I was unsure since I would have favored compactness, but it looks like there's a [`#replace` method](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/replace) we could use.

```suggestion
      document.documentElement.classList.replace('no-js', 'js');
```

(Would be good to update in the base template as well!)

Reply:
yeah +1 to your comment about how having a shared super simple base template would be nice

Reply:
Updated to `classList#replace` in 934eccc.